doc/articles/toc.yml

```diff
@@ -94,6 +94,8 @@
     - name: Tube Player
       topicHref: xref:Workshop.TubePlayer.Overview
       href: external/workshops/tube-player/toc.yml
+    - name: Chefs
+      href: external/uno.chefs/doc/toc.yml
```

Comment:
@kazo0, we are missing an overview page as well in the external/uno.chefs/doc/toc.yml

Reply:
https://github.com/unoplatform/uno.chefs/issues/1548

https://github.com/unoplatform/uno.chefs/issues/1549

Reply:
Those are done now :)